lighthouse-core/gather/drivers/extension.js

```diff
@@ -142,18 +141,16 @@ class ExtensionDriver extends Driver {
           return reject(chrome.runtime.lastError);
         }
 
-        this.url = tabs[0].url;
-        resolve(tabs[0].id);
+        resolve(tabs[0]);
```

Comment:
could be storing `this._tab` instead of `this._tabId` and just accessing the `id` prop when needed elsewhere